### ISSUE TITLE
Address.getAddressState() return null

### ISSUE THREAD

**Issue by wangzw** [TencentCloud/tencentcloud-sdk-java#3]
```
com.tencentcloudapi.vpc.v20170312.models.Address.getAddressState()  always return null.
```

From JSON response,  field `AddressState` should be serialized as `AddressStatus`

```
{
    "Response": {
        "TotalCount": 7,
        "AddressSet": [
            {
                "AddressId": "eip-96p3csg5",
                "AddressName": null,
                "AddressIp": "140.143.31.90",
                "AddressStatus": "UNBIND",
                "AddressType": "EIP",
                "InstanceId": null,
                "NetworkInterfaceId": null,
                "PrivateAddressIp": null,
                "IsArrears": false,
                "IsBlocked": false,
                "IsEipDirectConnection": false,
                "CreatedTime": "2018-05-11T16:01:51Z"
            },
           ......
        ],
        "RequestId": "3199c8ba-731f-4369-b95b-808c2d57f177"
    }
}
```

**Comment by zqfan** [comment]
谢谢您的反馈，我们已经将该问题记录并修正，下次更新时会修复这个问题。

**Comment by zqfan** [comment]
fixed in 3.0.4
